blockchain/message_pool/src/msgpool/msg_pool.rs

```diff
@@ -70,10 +73,17 @@ impl MsgSet {
 
     /// Add a signed message to the `MsgSet`. Increase `next_sequence` if the
     /// message has a sequence greater than any existing message sequence.
-    pub fn add(&mut self, m: SignedMessage) -> Result<(), Error> {
+    pub fn add(&mut self, m: SignedMessage, untrusted: bool) -> Result<(), Error> {
```

Comment:
What does `untrusted` mean? And it looks like you always set it to `false`.

Reply:
I did this for future endpoint (if we implement it). Right now we support only `Filecoin.MpoolPushMessage`.
But it exists a counterpart untrusted one: `Filecoin.MpoolPushUntrusted` for public gateways.

See lotus PR: https://github.com/filecoin-project/lotus/pull/3915